dotnet/src/SemanticKernel.Abstractions/Orchestration/SKContext.cs

```diff
@@ -25,6 +25,11 @@ public sealed class SKContext
     /// </summary>
     private CultureInfo _culture;
 
+    /// <summary>
+    /// Internal variables for Kernel use only.
+    /// </summary>
+    internal ContextVariables InternalVariables { get; }
```

Comment:
Is it expected this will still be around after all the phases are complete?

Reply:
This might go away if we have a way of passing parameters to ISKFunction that will not be part of the context (using attributes or something of like)